cl/users/tests.py

```diff
@@ -205,6 +205,9 @@ def test_deleting_your_account(self) -> None:
             reverse("delete_profile_done"),
         )
 
+    def test_generate_recap_email(self) -> None:
```

Comment:
Actually, this was supposed to be filled out as well. Just leaving a note for myself here.